ksql-benchmark/pom.xml

```diff
@@ -105,6 +105,22 @@ questions.
           <target>1.8</target>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>3.0.0-M1</version>
```

Comment:
any reason the version isn't in a properties section?

Reply:
Nope -- done.